_boards_cm/gumstix-pixhawk-fmuv6u-cm4.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: board
+title: "Gumstix Pixhawk FMUv6U + Pi CM4"
+short_description: This board incorporates a CM4, a Pixhawk FMUv6U compliant flight controller, and a Google Coral TPU.
+status: production
+picture: "/images/board-gumstix-pixhawk-fmuv6-cm4.jpg"
+github_issue: "https://github.com/geerlingguy/raspberry-pi-pcie-devices/issues/466"
+link: https://www.gumstix.com/manufacturer/cm4-pixhawk-fmuv6.html
+videos: [] IF NO VIDEOS
```

Comment:
This should just be `videos: []` in this case.

Reply:
Whoops. I made a pseudo-template for myself with some notes and failed to edit that out. Will re-submit.